src/main/java/org/opensearch/security/DefaultObjectMapper.java

```diff
@@ -287,12 +287,26 @@ public static TypeFactory getTypeFactory() {
         return objectMapper.getTypeFactory();
     }
 
+    @SuppressWarnings("removal")
     public static Set<String> getFields(Class<?> cls) {
```

Comment:
I have not seen any use of `getFields()` in the code of this PR. Wondering if the change below is necessary then.

Reply:
I remember encountering compilation issues which is why I had added. I have removed it for now.

Reply:
I was referring to the changes inside the method, especially the `AccessController.doPrivileged()` ... this still seems to be changed.